<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleXYScanWindow</class>
 <widget class="QMainWindow" name="SimpleXYScanWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>364</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Simple XY Scan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0" columnstretch="0,0,0">
    <item row="0" column="2" rowspan="2">
     <widget class="QGroupBox" name="details_groupBox">
      <property name="title">
       <string>Details</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Data</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QCheckBox" name="continuous_scan_checkBox">
         <property name="text">
          <string>Continuous</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="save_h5_checkBox">
         <property name="text">
          <string>Save H5</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QGroupBox" name="plot_groupBox">
      <property name="title">
       <string>Plot</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>View</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QCheckBox" name="show_previous_scans_checkBox">
         <property name="text">
          <string>Show 
Prev Scans</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="clear_previous_scans_pushButton">
         <property name="text">
          <string>Clear Prev</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="control_groupBox">
      <property name="title">
       <string>Parallelogram Scan Control:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="4">
        <widget class="QDoubleSpinBox" name="e1_x_doubleSpinBox"/>
       </item>
       <item row="3" column="4">
        <widget class="QDoubleSpinBox" name="e1_y_doubleSpinBox"/>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleSpinBox" name="p0_x_doubleSpinBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QDoubleSpinBox" name="e2_x_doubleSpinBox"/>
       </item>
       <item row="3" column="5">
        <widget class="QDoubleSpinBox" name="e2_y_doubleSpinBox"/>
       </item>
       <item row="4" column="5">
        <widget class="QDoubleSpinBox" name="e2_z_doubleSpinBox"/>
       </item>
       <item row="3" column="6">
        <widget class="QLabel" name="progres_label">
         <property name="text">
          <string>Progress:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>step</string>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QPushButton" name="interrupt_pushButton">
         <property name="text">
          <string>Interrupt</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QDoubleSpinBox" name="mag_1_min_doubleSpinBox"/>
       </item>
       <item row="5" column="6">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>num</string>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QDoubleSpinBox" name="mag_2_max_doubleSpinBox"/>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="start_pushButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="v_label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00aa00;&quot;&gt;mag 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QDoubleSpinBox" name="mag_1_max_doubleSpinBox"/>
       </item>
       <item row="7" column="5">
        <widget class="QDoubleSpinBox" name="mag_2_step_doubleSpinBox"/>
       </item>
       <item row="7" column="2">
        <widget class="QDoubleSpinBox" name="mag_2_min_doubleSpinBox"/>
       </item>
       <item row="5" column="4">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>max</string>
         </property>
        </widget>
       </item>
       <item row="4" column="6">
        <widget class="QDoubleSpinBox" name="progress_doubleSpinBox"/>
       </item>
       <item row="6" column="6">
        <widget class="QDoubleSpinBox" name="mag_1_num_doubleSpinBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>z</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="h_label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;mag 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="7" column="6">
        <widget class="QDoubleSpinBox" name="mag_2_num_doubleSpinBox"/>
       </item>
       <item row="6" column="5">
        <widget class="QDoubleSpinBox" name="mag_1_step_doubleSpinBox"/>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>min</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QDoubleSpinBox" name="e1_z_doubleSpinBox"/>
       </item>
       <item row="3" column="2">
        <widget class="QDoubleSpinBox" name="p0_y_doubleSpinBox"/>
       </item>
       <item row="4" column="2">
        <widget class="QDoubleSpinBox" name="p0_z_doubleSpinBox"/>
       </item>
       <item row="1" column="5">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#00aa00;&quot;&gt;e2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#0000ff;&quot;&gt;e1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#000000;&quot;&gt;p0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Scans a parallelogram spanned by (&lt;span style=&quot; font-weight:600;&quot;&gt;e1,e2&lt;/span&gt;) translated from the stage-origin by &lt;span style=&quot; font-weight:600;&quot;&gt;p0. &lt;/span&gt;The sidelengths are defined by the ranges mag_1 and mag_2 respectivly.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>mag_1_max_doubleSpinBox</tabstop>
  <tabstop>mag_1_step_doubleSpinBox</tabstop>
  <tabstop>mag_1_num_doubleSpinBox</tabstop>
  <tabstop>mag_2_max_doubleSpinBox</tabstop>
  <tabstop>mag_2_step_doubleSpinBox</tabstop>
  <tabstop>mag_2_num_doubleSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
